config/modules_statuses.json

```diff
@@ -1,10 +1,10 @@
 {
-    "Awards": true,
-    "Importer": true,
-    "Installer": true,
-    "Sample": true,
-    "Updater": true,
-    "VMSAcars": true,
-    "Vacentral": true,
-    "TestModule": true
-}
\ No newline at end of file
+  "Awards": true,
+  "Importer": true,
+  "Installer": true,
+  "Sample": true,
+  "Updater": true,
+  "VMSAcars": true,
+  "Vacentral": true,
+  "TestModule": true,
```

Comment:
Don't commit this file for now. I eventually will move these into the database

Reply:
Okay sure.
Thanks

Reply:
So, shall I delete and commit again?

Reply:
Yeah just remove it from the commit. I think a git rm --cached. That JSON file is also invalid - it's breaking the build (see travis below)

Reply:
Okay done.